scalardb-test/build.gradle

```diff
@@ -21,7 +24,7 @@ dependencies {
     if (project.hasProperty("scalardb-libs-path")) {
         implementation fileTree(project.property("scalardb-libs-path")) { include '*.jar' }
     } else {
-        implementation group: 'com.scalar-labs', name: 'scalardb', version: '3.5.0'
+        implementation group: 'com.scalar-labs', name: 'scalardb', version: '4.0.0-SNAPSHOT'
```

Comment:
Now that we release snapshot versions in Scalar DB, we can use them in kelpie-test.